be/src/olap/rowset/segment_v2/segment_iterator.cpp

```diff
@@ -1471,6 +1481,8 @@ Status SegmentIterator::_init_inverted_index_iterators() {
                     _segment->_tablet_schema->get_inverted_index(_opts.tablet_schema->column(cid)),
                     _opts, &_inverted_index_iterators[cid]));
         }
+        _inverted_index_iterators_by_col_name[_schema->column(cid)->name()] =
```

Comment:
We can use name -> cid mapping, but if that, we need two params in vexpr context, name -> id mapping, inverted index iterators list, instead of just _inverted_index_iterators_by_col_name